plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/xsd/com.google.cloud.tools.eclipse.appengine.libraries.xsd

```diff
@@ -108,6 +108,15 @@ limitations under the License.
                </xs:documentation>
             </xs:annotation>
          </xs:attribute>
+         <xs:attribute name="export" type="xs:boolean">
+            <xs:annotation>
+               <xs:documentation>
+                  If export attribute is true, then the corresponding library will be included in the WAR file of the
```

Comment:
What does `library.export=false` mean if a `libraryFile.export=true`.  Maybe add that this overrides any export settings on constituent libraryFile instances?

`libraryFile` should probably have `<tt>libraryFile</tt>`?